core/src/main/java/org/apache/iceberg/MetricsConfig.java

```diff
@@ -77,6 +79,14 @@ public static MetricsConfig getDefault() {
   }
 
   public static MetricsConfig fromProperties(Map<String, String> props) {
+    return from(props, null);
+  }
```

Comment:
Should we deprecate `fromProperties` or is there still a reasonable use for it? Looks like we're moving over all of our internal calls to `forTable`.

Reply:
Done

Reply:
I'll resolve this as it has been addressed.